### DISCUSSION TITLE
关于下载文件命名……

### DISCUSSION THREAD

**Discussion by Takenforgranted** [SPlayer-Dev/SPlayer#213] [想法 Ideas]
我觉得可以添加一个自定义下载的音乐文件的名称的功能，用户可以自定义选择使用 歌曲名 歌手名 专辑名 码率音质 等信息排列组合形成音乐文件的新名字。
这个功能感觉很多B站视频下载器都有……

**Comment by lionql** [comment]
原来有人已经提了，就是我之前的歌曲都是 '歌手-歌曲名'  这样命名的，但现在是歌曲名-歌手，还不能改，每一首下了都得改，好麻烦
